kie-server-parent/kie-server-remote/kie-server-rest/kie-server-rest-common/src/main/java/org/kie/server/remote/rest/common/resource/KieServerRestImpl.java

```diff
@@ -225,5 +231,44 @@ public Response updateReleaseId( @Context HttpHeaders headers,
     public Response getServerState(@Context HttpHeaders headers) {
         return createCorrectVariant(server.getServerState(), headers);
     }
+    
+    
+    
+    @ApiOperation(value="Readiness check for KIE Server that indicates that server is fully booted and ready to accept requests",
+            response=Void.class, code=200)
+    @ApiResponses(value = { @ApiResponse(code = 503, message = "Service not yet available") })
+    @GET
+    @Path("readiness")
```

Comment:
Would rename this to "readycheck" (same for method name)

Reply:
done